test/LocalizationWebsite/LocalizationWebsite.csproj

```diff
@@ -0,0 +1,26 @@
+<Project Sdk="Microsoft.NET.Sdk.Web">
+
+  <Import Project="..\..\build\common.props" />
+
+  <PropertyGroup>
+    <TargetFrameworks>netcoreapp1.1;net451</TargetFrameworks>
+    <!-- TODO remove when https://github.com/dotnet/sdk/issues/396 is resolved -->
+    <RuntimeIdentifier Condition=" '$(TargetFramework)' != 'netcoreapp1.1' ">win7-x64</RuntimeIdentifier>
+    <OutputType>Exe</OutputType>
+    <!-- workaround issues with satellite assembly generation and -->
```

Comment:
yes, that was the rest of the sentence. @ryanbrandenburg have you already opened a bug for this, or should I?